pkg/scheduler/algorithm/priorities/balanced_resource_allocation_test.go

```diff
@@ -215,7 +216,7 @@ func TestBalancedResourceAllocation(t *testing.T) {
 		pod          *v1.Pod
 		pods         []*v1.Pod
 		nodes        []*v1.Node
-		expectedList schedulerapi.HostPriorityList
+		expectedList extenderv1.HostPriorityList
```

Comment:
Hold off reviewing this for now. It will use `ScoreList` once #83260 gets merged. See https://github.com/kubernetes/kubernetes/pull/83262#issuecomment-536096681 for more details.